libraries/anaconda3-buildtime/context/build

```diff
@@ -2,9 +2,9 @@
 
 set -e
 
-CONDAENV=conda_environment
+CONDAENV=/tmp/conda_environment
```

Comment:
why tmp? I feel like temporary storage should be temporary. Let's put this in a more permanent location. I'm cool with that being somewhere in `/home/algo`

Reply:
works for me, /tmp is my default scratch space - but in this case /home/algo is fine (as long as the user is defined!?)